src/YQInputField.cc

```diff
@@ -253,8 +246,21 @@ void YQInputField::clearCapsLockWarning()
 }
 
 
+#if 0
 bool YQRawLineEdit::x11Event( XEvent * event )
 {
+    // FIXME: This used to check CapsLock, but it silently stopped working from
+    //        Qt 5.x on. Nobody ever realized this, but we want this feature
+    //        back some day soon.
+    //
+    // The reason why this never failed to compile was that this is overwriting
+    // a virtual function from the QWidget parent class, but the parent class
+    // simply doesn't have that function anymore, so this is just implementing
+    // a new function that is never called; so this failed silently.
+    //
+    // -- 2020-02-25 shundhammer
```

Comment:
Hehe, it's 2021 already ;)

Reply:
uhm, I completely missed that one :-)

Reply:
I guess it will take me until at least September to get used to this new number.
Why does it have to change so often? ;-)

Reply:
As time passes faster subjectively as you get older, I guess once you hit December it's not much worth borthering at all. :-)